.flowconfig

```diff
@@ -1,6 +1,3 @@
-[version]
-0.114.0
```

Comment:
Oh, we're removing this? I'm ok with that if we want!

Reply:
Yeah, I don't believe it's adding any value for the RRS and only makes updating flow harder.